src/linked_list.h

```diff
@@ -46,42 +46,42 @@ class SerialLinkedList : public LinkedList
     SerialLinkedList() : LinkedList() {}
     ~SerialLinkedList() {}
 
-    void Member() override;
-    void Insert(int data) override;
-    void Remove(int data) override;
+    int Member(int data) override;
```

Comment:
These 3 declarations should be virtual since the `MutexParallelLinkedList` and `RwLockParallelLinkedList` classes extend the `SerialLinkedList` class.